Makefile

```diff
@@ -1,5 +1,34 @@
-all: images
+TARGET_GOOS=linux
+TARGET_GOARCH=amd64
+TOOLS_BIN_DIR="build/_output/bin"
+
 .PHONY: all
+all: dist-gather-sysinfo images 
+
+.PHONY: clean
+clean: dist-clean
+
+.PHONY: dist-clean
+dist-clean:
+	rm -rf build/_output/bin
+
+.PHONY: dist
+dist: build-output-dir
+	@echo "Building ouput folder"
+	mkdir -p $(TOOLS_BIN_DIR); \
+
+.PHONY: build-output-dir
+build-output-dir:
+	mkdir -p $(TOOLS_BIN_DIR) || :
+
+.PHONY: dist-gather-sysinfo
+dist-gather-sysinfo: build-output-dir
+	@if [ ! -x $(TOOLS_BIN_DIR)/gather-sysinfo ]; then\
+		echo "Building gather-sysinfo helper";\
+		env CGO_ENABLED=0 GOOS=$(TARGET_GOOS) GOARCH=$(TARGET_GOARCH) go build -ldflags="-s -w" -mod=vendor -o $(TOOLS_BIN_DIR)/gather-sysinfo ./pkg/tools/gather-sysinfo/;\
```

Comment:
Why you're building it? From the email thread it looked like you already have a package ready that you just install?

Reply:
I do not think we have ever said that. We are currently building it as part of the separate must gather image we ship, but there never was an RPM as far as I know.